### DISCUSSION TITLE
How do I hide this version drop down?

### DISCUSSION THREAD

**Discussion by pateash** [lepture/shibuya#57] [Q&A] [answered]
<img width="1663" alt="image" src="https://github.com/lepture/shibuya/assets/16856802/f5c98c7d-9aab-40c0-9909-c53176aa5fcd">
the version drop down on top having wrong versions sometimes even though I have disabled/hidden that version in readthedocs.

**Comment by lepture** [comment, marked as answer]
Create an empty file at your `docs/source/_templates/components/nav-versions.html`.
